cmd/juju/application/deployer/charm.go

```diff
@@ -258,7 +258,12 @@ func (d *deployCharm) deploy(
 	}
 
 	if errors.IsAlreadyExists(err) {
-		return errors.Wrapf(err, errors.Errorf("\ndeploy application using an alias name, or use remove-application to remove the existing one and try again"), err.Error())
+		// Would be nice to be able to access the app name here
+		return errors.Wrapf(err, errors.Errorf(`
```

Comment:
Remove extra new line

Reply:
In the error message? It's there to format things nicely

Reply:
Previously, when I've done this, I move the error message outside of the function scope and into a const at the package level.